notebook2/notebook/notebook.py

```diff
@@ -111,11 +119,46 @@ def get_name(line):
         "containing the name of the docker image to use for worker pods.") from e
 
 
+def jwt_decode(token):
```

Comment:
While you were away I added `/hailjwt`, it's more or less what you have here except I added some tests and required the token is a string or bytes. Can we rebase on master and use that library?

We'll need a line in the makefile that does something like `cd ../hailjwt && python setup.py install`

Reply:
I'm working on a PR that uses it for batch